src/dev/license_checker/config.ts

```diff
@@ -30,7 +30,7 @@
 
 // The following list applies to packages both
 // used as dependencies or dev dependencies
-export const LICENSE_WHITELIST = [
+export const LICENSE_ALLOWLIST = [
```

Comment:
I'm still confused with this change. As we are exporting these ones, is it safe to rename or should we be deprecating old one and add the new ones?  same at line #89. @kavilla can you confirm?

Reply:
I think license checker is in src/dev. It won’t be build to artifacts and shouldn’t cause some deprecation/bwc issue. So I think it is fine.